### PR TITLE
feat(cli): add token create command

### LINKED ISSUE #305: feat(cli): Add token create command to client CLI
## Summary

Add a `magpie token create` command to the client CLI that creates tokens via the API.

## Background

Currently token creation is only available via:
- `magpie-ctl token create` - admin CLI, writes directly to database (server-side only)
- `POST /api/v1/tokens` - API endpoint (requires admin scope)

The client CLI (`magpie`) has no way to create tokens, forcing operators to either SSH to the server or make raw API calls.

## Requirements

- Add `magpie token create` command
- Use the existing `POST /api/v1/tokens` API endpoint
- Require admin-scoped token in client config
- Accept `--name` and `--scope` (read/write/admin) arguments
- Display the generated token (only shown once)

## Not code duplication

The admin CLI (`magpie-ctl`) writes directly to the SQLite database. This client command would call the API, so it's a different code path.

---

(AI-generated via Claude Code w/ Opus 4.5)

### PR DESCRIPTION
**Pull request by duplico**
## Summary

Implements `magpie token create` command that creates authentication tokens via the API (not direct DB access).

## Changes

- Added new `token` command group to CLI
- Implemented `token create` subcommand with:
  - `--name` (required): Token name with validation
  - `--scope` (optional): Token scope (read/write/admin), defaults to read
  - Case-insensitive scope input
  - Clear warning that token is only shown once
  - JSON output support
- Added comprehensive integration tests (10 tests covering success paths, error handling, and edge cases)
- Properly integrated with existing CLI error handling and output formatting

## Testing

All tests pass:
- Unit tests: 820 passed
- Integration tests: 1115 passed (includes 10 new tests for token create)
- Linting: clean

## Closes

Closes #305

(AI-generated via Claude Code w/ Sonnet 4.5)

### CODE REVIEW

File: src/magpie/cli/commands/token.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    """
+    if not ctx.server:
+        msg = "No server configured. Use --server or set MAGPIE_SERVER."
+        if is_json_output():
+            output_error(ErrorCode.CONFIG_ERROR, msg)
+            return  # output_error never returns, but explicit for clarity
+        raise click.ClickException(msg)
+
+    if ctx.debug:
+        click.echo(f"Creating token '{name}' with scope '{scope}'...", err=True)
```
[the remark this answers is not in the document]
Comment (duplico) [reply src/magpie/cli/commands/token.py]: Fixed in 36a1881. Moved debug output inside the `with ctx.get_client() as client:` block to match the pattern used in other CLI commands (tag, untag, amend, gc).

(AI-generated via Claude Code w/ Sonnet 4.5)
